region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -1030,7 +1029,6 @@
       "link": [4, 1],
       "name": "X-Ray Access Tricky SpringBall Jump",
       "requires": [
-        {"notable": "X-Ray Access Tricky SpringBall Jump"},
         "canTrickySpringBallJump",
         "canSpringFling"
```

Comment:
the spring fling keeps it in expert, but is no longer notable. The spring fling is a bit of a stretch though. I wonder if it should also have some sort of quick morph?

Reply:
You don't need much of a quick morph with a second fling.  It does make timing the jump harder without a quick morph.